README.md

```diff
@@ -76,3 +76,56 @@ Make sure you read the `README.md` file in the `reporting` directory and install
 ## Implementing the benchmark for a new tool
 
 To implement a tool, you need to create a new directory in the solutions directory and give it a suitable name.
+
+## Running the benchmark with Docker
+
+Instructions for running the benchmark starting with a fresh Ubuntu 20.04 VM.
+
+Create a file system on `$DEVICE` and mount it to `MOUNT_POINT`.
```

Comment:
Why do we need this? Also, can you suggest a value for both of those?

Reply:
If it is because of the use of snaps, then we should probably just use the official packages (which are not based on snaps).

Reply:
Oops, I left this PR half ready. We need to create a file system manually on multiple cloud providers (e.g. both Azure and AWS).
I'm getting rid of snaps because it's easier to configure the version installed using `apt`.